vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -17,11 +17,11 @@ NODE_DRAIN_TIMEOUT: "{{ lookup('env', 'NODE_DRAIN_TIMEOUT') | default('0s', true
 BMOPATH: "{{ lookup('env', 'BMOPATH') }}"
 IRONIC_DATA_DIR: "{{ lookup('env', 'IRONIC_DATA_DIR') }}"
 KUBECONFIG_PATH: "/home/ubuntu/.kube/config"
-KUBERNETES_VERSION: "{{ lookup('env', 'KUBERNETES_VERSION') | default('v1.21.0', true) }}"
-UPGRADED_K8S_VERSION: "{{ lookup('env', 'UPGRADED_K8S_VERSION') | default('v1.21.1', true) }}"
+KUBERNETES_VERSION: "{{ lookup('env', 'KUBERNETES_VERSION') | default('v1.21.1', true) }}"
```

Comment:
You are uplifting the default here only but since you havent changed the  `KUBERNETES_VERSION` in `lib/common` whats the point of changing default only here?

Reply:
I agree.